src/app/components/notes/notes.component.ts

```diff
@@ -46,12 +46,11 @@ export class NotesComponent implements OnInit {
     this.newNoteOpened = !this.newNoteOpened;
   }
 
-  refreshSubject(subject:string){
-    this.onSelectSubject(subject)
+  refreshSubject(subject: string) {
+    this.onSelectSubject(subject);
   }
-  
+
   anyNotesCheck(): boolean {
```

Comment:
Is the "anyNotesCheck" method used somewhere?

Reply:
Not really. Removing it.